src/styles.css

```diff
@@ -34,6 +34,7 @@
     transform: translateX(0) scale(1);
 }
 
-pre[v-pre]:hover .docsify-copy-code-button {
+.docsify-copy-code-button:focus,
```

Comment:
nice touch

Reply:
Stumbled on it while building [docsify-tabs](https://jhildenbiddle.github.io/docsify-tabs). The focus ring kept disappearing when trying to navigate the tabs with the keyboard, then I realized the "Copy Code" button has focus, but its opacity was 0.